jest.config.mjs

```diff
@@ -0,0 +1,9 @@
+export default {
+  verbose: true,
+  testMatch: ['<rootDir>/test/**/*.?js'],
+  transform: {},
+  moduleFileExtensions: ["js", "jsx", "mjs"],
+  testPathIgnorePatterns: ["node_modules", "__mocks__", "support"],
```

Comment:
**annotation**:
Jest isn't smart enough not to try to parse its own required `__mocks__` directory as tests, so we have to explicitly ignore it here unless we want to put mocks at the base of the project.